Fatal error: exception Failure("funner already declared")